### PR TITLE
Fixes #708 "Added option for donation"

### LINKED ISSUE #708: Add a donation button "Support Open Food Facts"
image
image
image

### PR DESCRIPTION
**Pull request by ManognaMutyala**
## Description

Describe the changes made and why they were made instead of how they were made.
Added option for donation

## Related issues and discussion
#708 
 

 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

File: app/src/main/res/xml/preferences.xml [hunk 1]
```diff
@@ -26,6 +26,12 @@
         android:key="Locale.Helper.Selected.Language"
         android:summary="@string/prefs_language_summary"
         android:title="@string/prefs_language" />
+    <Preference
+        android:summary="@string/Donation"
+        android:title="@string/Donate">
+        <intent android:action="android.intent.action.VIEW"
+            android:data="https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en" />
```
Comment (policorn) [inline app/src/main/res/xml/preferences.xml]: Please use a `@string` resource for the URL so it can be internationalized

Comment (ManognaMutyala) [reply app/src/main/res/xml/preferences.xml]: Ok @policorn, I will definitely make the changes

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -114,6 +114,8 @@
     <string name="txtStores">Stores :</string>
     <string name="txtCountries">Countries where sold :</string>
     <string name="txtIngredients">Ingredients list :</string>
+    <string name="Donate">Donate</string>
+    <string name="Donation">Click here to donate for Open Food facts</string>
```
Comment (PrajwalM2212) [inline app/src/main/res/values/strings.xml]: "Click here to donate for Open Food facts" 
Can you change the string to "Click here to donate for Open Food Facts" ?

Comment (Karljoones) [reply app/src/main/res/values/strings.xml]: It should actually be "Click here to donate to Open Food Facts"

**Comment by teolemon** [comment]
Fixed the string. Thanks @PrajwalM2212

**Pull request merged**